src/test/java/BoardTest.java

```diff
@@ -19,7 +19,7 @@ public void testAddAMarkToBoard() {
     }
 
     @Test
-    public void testKnowThatAMoveIsValid() {
+    public void testKnowsThatAMoveIsValid() {
```

Comment:
I dislike the test* prefix on tests, this used to be a thing because old-school JUnit3 required you to do it, but I think it just gets in the way in modern JUnit tests.

Reply:
✅ PR #6